### PR TITLE
refactor: Integrate logging package

### PR DESCRIPTION
**Pull request by ozayr-zaviar**
## Summary
- Integrate logging package in `optimizely-sdk` package.

## Test plan
- All unit tests and Full stack compatibility suite tests pass.

## Jira
[OASIS-8204](https://optimizely.atlassian.net/browse/OASIS-8204)

### CODE REVIEW

**Review by zashraf1985 — CHANGES_REQUESTED** [review]
Looks Great! Added a question. Also please double check all the copyright header changes too.

File: packages/optimizely-sdk/lib/index.browser.ts [hunk 1]
```diff
@@ -13,14 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import logHelper from './modules/logging/logger';
```
[the remark this answers is not in the document]
Comment (ozayr-zaviar) [reply packages/optimizely-sdk/lib/index.browser.ts:16]: Karma browser test was not able to locate the package unless it is defined in the default.

**Review by zashraf1985 — APPROVED** [review]
LGTM! @opti-jnguyen @mikechu-optimizely it will be helpful if you guys can also take a look and provide feedback on this.

File: packages/optimizely-sdk/lib/index.browser.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (opti-jnguyen) [reply packages/optimizely-sdk/lib/index.browser.ts:16]: This comment applies to most of the files below that import the logging module as both `logging` and `logging/logger`.

Let me know if I'm misunderstanding something.

**Comment by opti-jnguyen** [comment]
Added a couple comments @ozayr-zaviar - let me know what you think.

cc: @zashraf1985

File: packages/optimizely-sdk/lib/index.browser.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (zashraf1985) [reply packages/optimizely-sdk/lib/index.browser.ts:16]: I asked the same question and looks like he is having some trouble using these in tests when he directly imports these. Lets me try debugging it with uzair together and see if we can simplify this.

**Review by opti-jnguyen — APPROVED** [review]
Minus the small non-blocking notes from my comments, LGETM!

**Pull request merged**